Neos.ContentRepositoryRegistry/Classes/Command/CrCommandController.php

```diff
@@ -33,18 +32,36 @@ public function __construct(
      * Note: This command is non-destructive, i.e. it can be executed without side effects even if all dependencies are up-to-date
      * Therefore it makes sense to include this command into the Continuous Integration
      *
+     * To check if the content repository needs to be setup look into cr:status.
+     * That command will also display information what is about to be migrated.
+     *
      * @param string $contentRepository Identifier of the Content Repository to set up
+     * @param bool $resetProjections Advanced. Can be used in rare cases when the projections cannot be migrated to reset everything in advance. This requires a full replay afterwards.
      */
-    public function setupCommand(string $contentRepository = 'default'): void
+    public function setupCommand(string $contentRepository = 'default', bool $resetProjections = false): void
     {
         $contentRepositoryId = ContentRepositoryId::fromString($contentRepository);
+
+        if ($resetProjections) {
+            if (!$this->output->askConfirmation(sprintf('> Advanced Mode. The flag --reset-projections will reset all projections in "%s", which leaves you with empty projections to be replayed. Are you sure to proceed? (y/n) ', $contentRepositoryId->value), false)) {
```

Comment:
As `cr:setup` can be run in the CI didn't we also expect a `--force` option here 🤔?

Reply:
Yes and no. The advanced mode `--reset-projections` is NOT expected to be run on ci. It should in general be not used. Only by us.

So a normal `cr:setup` will do just fine as before.